package.json

```diff
@@ -21,14 +21,16 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-games#readme",
   "dependencies": {
+    "express": "^4.18.1",
+    "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
-    "pg-format": "^1.0.4",
-    "husky": "^7.0.0"
+    "pg-format": "^1.0.4"
   },
   "devDependencies": {
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "pg": "^8.7.3",
+    "supertest": "^6.2.3"
   },
```

Comment:
do you need this?

Reply:
I just did npm install and then installed express and supertest as it said so in the the second ticket. May have a mistake somewhere though